packages/server/modules/user-emails/domain/operations.ts

```diff
@@ -0,0 +1,22 @@
+import { UserEmail } from './types'
```

Comment:
Do user emails really deserve their own speckle module? There's not much going on in there